src/aleph_client/utils.py

```diff
@@ -100,3 +101,33 @@ def extract_valid_eth_address(address: str) -> str:
         if match:
             return match.group(0)
     return ""
+
+
+async def list_unlinked_keys() -> Tuple[List[Path], Optional[ChainAccount]]:
+    """
+    List private key files that are not linked to any chain type and return the active ChainAccount.
+
+    Returns:
+        - A tuple containing:
+            - A list of unlinked private key files as Path objects.
+            - The active ChainAccount object (the single account in the config file).
+    """
+    config_home = settings.CONFIG_HOME if settings.CONFIG_HOME else str(Path.home())
```

Comment:
You probably want to convert everything to a Path instead of converting a Path to a str to convert it back to a Path after that.

But that's really a minor detail.